web/admin/src/request/types.ts

```diff
@@ -1340,8 +1342,9 @@ export interface V1NotionParseResp {
 }
 
 export interface V1NotionScrapeReq {
-  doc_id?: string;
-  id?: string;
+  doc_id: string;
```

Comment:
在 `V1NotionScrapeReq` 接口中，`doc_id` 和 `id` 字段从可选变为了必填。请确认这是否符合后端接口的实际要求，如果后端接口仍然接受可选字段，则此处修改可能会导致不必要的限制。

```suggestion
// 如果后端接口支持可选字段，请保持一致性
doc_id?: string;
id?: string;